src/main/scala/com/simple/jdub/Transaction.scala

```diff
@@ -26,6 +26,38 @@ class Transaction(connection: Connection) extends Queryable {
     onRollback.foreach(_())
   }
 
+  /**
+   * Roll back the transaction to a savepoint.
+   */
+  def rollback(savepoint: Savepoint) {
```

Comment:
It is a "future" deprecation at this point, but I agree it should be changed. How would you feel about changing every procedure syntax in the entire project in a separate PR? This matches the surrounding code style and I'd prefer not to litter this PR with syntactic changes to a bunch of other code.